src/inc/StreamHelper.hpp

```diff
@@ -28,5 +30,24 @@ namespace MSIX {
             ThrowHrIfFailed(stream->Seek(start, StreamBase::Reference::START, nullptr));
             return buffer;
         }
+
+        inline std::pair<std::uint32_t, std::uint8_t*> CreateRawBufferFromStream(const ComPtr<IStream>& stream)

```

Comment:
>std::pair<std::uint32_t, std::uint8_t*> [](start = 15, length = 39)

Are you shying away from a std::vector for a reason?  Does it bloat the binary?

Reply:
talked offline, this will change to std::pair<std::uint32_t, std::unique_ptr<std::uint8_t>>

---
In reply to: [273627988](https://github.com/Microsoft/msix-packaging/pull/82#discussion_r273627988) [](ancestors = 273627988)